_mydocs/calendar-pro/options/granularity/group.md

```diff
@@ -6,6 +6,9 @@ date:               2022-07-27
 modified:           2022-07-27
 order:              /calendar-pro/options/calendar-granularity/group-by
 internal:           group
-draft:              true
 ---
-**Default value:** Auto
\ No newline at end of file
+The **Group** parameter manages how data are grouped in the visual.
+
+The **Default value** is **Auto**.
```

Comment:
Please leave **Default value:** XXX
We used this label everywhere for other visuals (e.g. Bullet Chart options).
Can you please revert this change here and in other places where you changed it?

Reply:
Shure! Working on that right now. :)